.github/workflows/pr.yaml

```diff
@@ -10,6 +10,24 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  changed-files:
+    runs-on: ubuntu-latest
+    name: "Check changed files"
+    outputs:
+      test_cpp: ${{ steps.changed-files.outputs.ci_any_changed == 'true' || steps.changed-files.outputs.cpp_any_changed == 'true' }}
+      test_python: ${{ steps.changed-files.outputs.ci_any_changed == 'true' || steps.changed-files.outputs.cpp_any_changed == 'true' || steps.changed-files.outputs.python_any_changed == 'true' }}
+    steps:
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v45
+        with:
+          files_yaml: |
+            ci:
+              - ci/**
+            cpp:
```

Comment:
changes to root level `rapids_config.cmake`, and `build.sh` need to trigger tests